libs/blocks/section-metadata/section-metadata.js

```diff
@@ -42,4 +47,5 @@ export default function init(el) {
   if (metadata.style) handleStyle(metadata.style.text, section);
   if (metadata.background) handleBackground(metadata.background.content, section);
   if (metadata.layout) handleLayout(metadata.layout.text, section);
+  if (metadata.sticky) handleSticky(metadata.sticky.text, section);
```

Comment:
Not sure what was said in the other PR, but couldn't you just add `sticky-bottom`, `sticky-top` in section-metadata under styles? I understand if you want this to be explicit for AX purposes though.

Reply:
Great idea @JasonHowellSlavin

Reply:
Not a bad idea. I guess that would help with not "junking up" `section-metadata` with more properties. Sounds good to me.